core/server/data/migrations/versions/4.9/01-add-default-product-portal-products.js

```diff
@@ -19,17 +19,8 @@ module.exports = createTransactionalMigration(
             .select('value')
             .first();
 
-        if (!portalProductsSetting) {
-            logging.info('Skipping adding default product to portal_products, setting does not exist');
-            return;
-        }
-
         const portalProducts = JSON.parse(portalProductsSetting.value);
 
-        if (portalProducts.length > 0) {
```

Comment:
:thinking: I'm not sure that this check is redundant, without it I don't think the migration is idempotent, as if it were to run twice we would go from `[] -> [productId] -> [productId, productId]`

Reply:
🤔 Hmm good point, have added this back now, thanks.